modules/ROOT/pages/create-profiles.adoc

```diff
@@ -1,7 +1,7 @@
 = Applying Governance Rulesets to Specified APIs
 
 After you identify the APIs in Exchange that 
-you want to govern, you are ready to create governance profiles. Governance profiles enable you to select the specific sets of APIs that you want to govern for each set of rulesets.
+you want to govern, you are ready to create governance profiles. Governance profiles enable you to filter the specific sets of APIs to govern for each set of rulesets.
```

Comment:
```suggestion
you want to govern, you can create governance profiles. Governance profiles enable you to filter the specific sets of APIs to govern for each set of rulesets.
```

Reply:
I made a slightly different change, since "you can" implies it is a requirement to identify them first, whereas it is reallyl a best practice.